sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-jedis/src/main/java/com/azure/messaging/eventhubs/checkpointstore/jedis/JedisRedisCheckpointStore.java

```diff
@@ -34,7 +34,11 @@ public class JedisRedisCheckpointStore implements CheckpointStore {
     static final byte[] PARTITION_OWNERSHIP = "partitionOwnership".getBytes(StandardCharsets.UTF_8);
     private final JedisPool jedisPool;
 
-    JedisRedisCheckpointStore(JedisPool jedisPool) {
+    /**
+     * Constructor for JedisRedisCheckpointStore
+     * @param jedisPool a JedisPool object that creates a pool connected to the Azure Redis Cache
+     */
+    public JedisRedisCheckpointStore(JedisPool jedisPool) {
         this.jedisPool = jedisPool;
```

Comment:
Should check for Null since this is exposed to the customer... and add javadocs to throw.